query/src/sessions/session_status.rs

```diff
@@ -72,6 +74,18 @@ impl MutableStatus {
         *lock = db
     }
 
+    // Set the current user after authentication
```

Comment:
It would be nice to add a unit test in test crate we have some MutableStatus tests there

Reply:
@BohuTANG thank you for remind, i've added a test case on the getter/setter for current user

Reply:
to test the behaviour of session's current user, i'd split another PR to add a current_user() function